main.cpp

```diff
@@ -131,27 +186,27 @@ int main()
     auto carRented = rentACar(6, 2); 
     
     //1)
-    
+    auto selectedShoe = ShoeSelection(12, 2);
     //2)
-    
+    auto punchCard = timeSheet (9, 0, 18, 30);
     //3)
-    
+    auto notePlayed = playANote (36, 64, 16);
```

Comment:
your params need to be `char` types, not integers.  you're forcing an implicit conversion from `int` to `char`
- [ ] revise